sdks/python/server/README.md

```diff
@@ -0,0 +1,340 @@
+# MCP UI Server SDK for Python
+
+A Python SDK for creating MCP UI resources on the server side, enabling rich interactive experiences in MCP applications.
+
+## Installation
+
+```bash
+pip install mcp-ui-server
+```
+
+## Quick Start
+
+```python
+from mcp_ui_server import create_ui_resource
+
+# Create an HTML resource
+resource = create_ui_resource({
```

Comment:
Note that the README uses `create_ui_resource`, unlike the current implementation (`create_UIResource`)